test-app/src/main/resources/application.yml

```diff
@@ -59,20 +59,3 @@ management:
 oauth:
   endpoint.url: http://localhost:8090/auth
 
-# These should not be required in your main application and should instead be provided by helm values referencing Kuberentes secrets - these are only here for unit tests
```

Comment:
Removed the unit test as the test-app is a kind of living example application but in real consumers of the library there should not be any config required in application.yml